src/components/nodes/callActivity/callActivity.vue

```diff
@@ -67,6 +67,7 @@ export default {
   },
   mounted() {
     this.shape = new TaskShape();
+    const defaultName = this.node.definition.get('name');
```

Comment:
I think of "default" as a fall-back value to use when the user supplied or expected value doesn't exist. But if this process is being loaded in spark, then what gets returned from the definition may not be a default value, but something the user initially supplied.
 
If you're aiming to increase readability, I think something like `callActivityName` may be better here.